agora.md

```diff
@@ -0,0 +1,288 @@
+
+<p align="center">
+<img src="/assets/img/agora-buying-bitcoin-01.png" class=responsive style="max-width: 100%; height: auto;" />
+</p>
+
+If the payment methods provided do not meet your needs, you have the option to create an advertisement featuring your preferred payment method. Learn more [here](https://agoradesk.com/nojs/ad/new). 
+
+**Later, we will see how to create our own offers!**
+
+Before accepting a satisfying offer, make sure to read the "Terms of the Trade" section. Be cautious of trade partners who demand personal information such as ID photos under the pretense of "security" measures. Such offers contradict the objective of getting non-KYC Bitcoin.
+
+<p align="center">
+<img src="/assets/img/agora-buying-bitcoin-02.png" class=responsive style="max-width: 100%; height: auto;" />
+</p>
+
+You can also check your trade partner's reputation by clicking their username. This will reveal their feedback score - the higher, the better - as well as reviews from other users.
+
+Colored circles next to a trader's username indicate their recent activity: green means they've been active today, yellow points to activity within the week, and a gray circle shows no site activity for over a week.
+
+<p align="center">
+<img src="/assets/img/agora-buying-bitcoin-04.png" class=responsive style="max-width: 100%; height: auto;" />
+</p>
+
+To initiate a trade, enter the amount you wish to purchase, provide a new address from your wallet (avoid using the same address twice), and click on "Send Trade Request".
+
+<p align="center">
+<img src="/assets/img/agora-buying-bitcoin-03.png" class=responsive style="max-width: 100%; height: auto;" />
+</p>
+
+After sending your trade request, wait for the seller to accept your offer. Upon acceptance, you will receive the payment details via the encrypted chat function on the trade page (The chat is accessible to you and your trade partner). 
+
+**Please note that Agora retains chat messages for 180 days for dispute resolution purposes.**
+
+After making payment according to the seller's instructions, confirm by clicking "I have paid". This will notify the seller about the successful payment, ensuring the trade won't be canceled.
+
+<p align="center">
+<img src="/assets/img/agora-buying-bitcoin-05.png" class=responsive style="max-width: 100%; height: auto;" />
+</p>
+
+As the payment is confirmed, the trade settlement process starts. The status will switch to "Processing". 
+
+Now, wait for the coins to be automatically transferred to your provided wallet address. This process can take between 10 to 60 minutes, with network fees being deducted from the final amount, so you might receive slightly less than anticipated.
```

Comment:
Is this bit automatic, or is it dependent on the sender confirming receipt of payment?

Reply:
When the seller confirms receipt of payment, the sats are automatically released. I will reword it to make it clearer. 👍